cob_srvs/CMakeLists.txt

```diff
@@ -6,12 +6,13 @@ find_package(catkin REQUIRED COMPONENTS geometry_msgs message_generation)
 add_service_files(FILES
   Dock.srv
   SetFloat.srv
+  SetHornMode.srv
   SetInt.srv
   SetString.srv
 )
 
 generate_messages(
-  DEPENDENCIES geometry_msgs
+  DEPENDENCIES geometry_msgs cob_msgs
```

Comment:
sort alphabetical

Reply:
Done!

Reply:
Should I combine the fixes to one commit and do a force push?

Reply:
yes, that would make sense